torch/csrc/utils/python_dispatch.cpp

```diff
@@ -688,6 +688,16 @@ void initDispatchBindings(PyObject* module) {
         return names;
       },
       py::arg("dispatch_key") = static_cast<const char*>(""));
+  m.def(
+      "_dispatch_set_report_error_callback",
+      [](c10::OperatorHandle& handle, py::object callback) {
+        auto obj = callback.ptr();
+        // SafePyObject steals a reference
+        Py_INCREF(obj);
```

Comment:
It's more idiomatic to release() into SafePyObject, saves you an extra decref on exit